src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -579,4 +555,8 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     return false;
   }
 
+  private HostAndPort generateHostAndPort(List<Object> hostInfos) {
```

Comment:
@HeartSaVioR I believe it's used here https://github.com/xetorthio/jedis/pull/1204/files#diff-834ad1f566a8da02a00da75ce4e9c3a6R69